django/middleware/locale.py

```diff
@@ -16,28 +16,35 @@ class LocaleMiddleware(MiddlewareMixin):
 
     response_redirect_class = HttpResponseRedirect
 
+    def get_fallback_language(self, request):
+        # Allows for easier fallback logic by extending LocaleMiddleware
+        # and overriding this method. Fallback to supported languages
+        # if a variant is used
```

Comment:
If it's worth writing a comment for it may be worth turning it into a docstring (with the phrasing to suite)